### PR TITLE
Tags editor, usability improvements: MK5

### PR DESCRIPTION
**Pull request by eshurakov**
### Fix
In this PR we're adding the following to the tags editor:
* RTL support
* Add tag field stays visible when active

Both issues were mentioned in https://github.com/Automattic/simplenote-ios/pull/1149#issuecomment-779874151

Ref: #1067

### Test
- [ ] Add tag field is always visible when you enter new tags
- [ ] When device's locale is RTL, tags are pinned to the right side and ordered right-to-left

### Review
Only one developer is required to review these changes, but anyone can perform the review.

### CODE REVIEW

File: Simplenote/Classes/TagView.swift [hunk 1]
```diff
@@ -128,7 +128,10 @@ class TagView: UIView {
             return
         }
 
-        scrollView.scrollRectToVisible(addTagField.frame.insetBy(dx: -Constants.wrappingStackViewMargins.right, dy: 0),
+        layoutIfNeeded()
```
Comment (jleandroperez) [inline Simplenote/Classes/TagView.swift:131]: This invocation here _feels_ off. But TBH I'm not sure where it could be otherwise!!

**Review by jleandroperez — APPROVED** [review]
@eshurakov thank you for fixing this!!

:shipit:

File: Simplenote/Classes/TagView.swift (hunk 1 above)
Comment (eshurakov) [reply Simplenote/Classes/TagView.swift:131]: Totally agree on that 🤔

**Comment by eshurakov** [comment]
Thank you!

**Comment by pachlava** [comment]
[testing] fix/feature verified on:

model | iOS | Simplenote
-|-|-
iPhone XR | 14.4 | 4.33.0.0
iPad Pro 11” Sim | 14.4 | 4.33.0.0

**Pull request merged**